app/src/components/organisms/SurveyClassQuestion.tsx

```diff
@@ -0,0 +1,106 @@
+import React, { useState } from 'react';
+import Radio from '@mui/material/Radio';
+import RadioGroup from '@mui/material/RadioGroup';
+import FormControlLabel from '@mui/material/FormControlLabel';
+import FormControl from '@mui/material/FormControl';
+import FormLabel from '@mui/material/FormLabel';
+import Stack from '@mui/material/Stack';
+import { willingRadio, ability, willing } from '../shared/constants/surveyForm.constants';
+
+function SurveyClassQuestion(props: {
+  name: string;
+  fieldChanged: (fieldId: string, type: string, value: string) => void;
+}) {
+  const [disabled, setDisabled] = useState(true);
+  const [checked, setChecked] = useState({ willing: false, notWilling: false, veryWilling: false });
+
+  const handleDisabling = (event: React.ChangeEvent<HTMLInputElement>) => {
+    if (event.target.value === ability.cannot) {
+      setDisabled(event.target.checked);
+
+      setChecked(() => {
+        return willingRadio;
+      });
+    } else {
+      setDisabled(!event.target.checked);
+      setChecked(() => {
+        return {
+          ...willingRadio,
+          notWilling: true
+        };
+      });
+    }
+    props.fieldChanged(props.name, 'ability', event.target.value);
+  };
+
+  const changeRadio = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setChecked(() => {
+      return {
+        ...willingRadio,
+        [event.target.value]: true
+      };
+    });
+
+    props.fieldChanged(props.name, 'willing', (event.target as HTMLInputElement).value);
+  };
+
+  return (
+    <Stack key={props.name} direction="row" spacing={5}>

```

Comment:
UI is a good start but I think we would benefit by cutting down on the vertical space for each row. I would add a new column to the left that contains the course number (ie. CSC 423). Then, add headers to the grids for Course number | Teaching ability | Willingness to teach

Reply:
I agree - spacing={2} might be a cleaner look

Reply:
The spacing for each row is in the surveyForm component and is now decreased to 2 from 4